swift/apple/FirezoneKit/Sources/FirezoneKit/Views/MenuBar.swift

```diff
@@ -42,6 +43,7 @@
       self.statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.variableLength)
 
       self.appStore = appStore
+      self.tunnelStore = appStore.tunnelStore
```

Comment:
MenuBar seems to have an implicit dependency on `tunnelStore`, so I made it explicit.